core/src/main/scala/chisel3/Choice.scala

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.experimental.{BaseModule, SourceInfo}
+import chisel3.util.simpleClassName
+
+/** This package contains Chisel language definitions for describing configuration options and their accepted values.
+  */
+package object choice {
```

Comment:
Stylistically, this file should be named: `core/src/main/scala/chisel3/choice/package.scala`